wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,260 @@
-Song Setup
-============
+# Song Setup
 
-The Song Setup is an important screen in [beatmapping](/wiki/Beatmapping), as it contains many important settings that are keys to make a decent map.
 
-![Where the Song Setup is located](img/Song_Setup_nav.jpg "Where the Song Setup is located")
+Song Setup is the fourth section of the [Beatmap Editor](/wiki/Beatmap_Editor). It contains metadata and general beatmap configuration settings.
 
-General
--------
+## General
 
-### Song and Map Meta Data
+### Song and Map Metadata
 
-![An example of complete data (General tab)](img/Song_Setup.jpg "An example of complete data (General tab)")
+*See also: [\[Guide\] Metadata](https://osu.ppy.sh/community/forums/topics/249288 "osu! forums")*
 
-[Guide on finding Metadata](https://osu.ppy.sh/forum/t/249288).
+![A completed song setup form](img/SS_General.jpg "An example of complete metadata (General tab) in Song Setup")
 
-Not every song needs something in the "Source" field. The source is only for songs that are "from" something, such as a video game theme song, or something from the soundtrack of an anime. Regular rock and pop songs can just leave the source field blank.
+These fields should be completed before creating a second difficulty. This is primarily to ensure that the metadata will be consistent across all difficulties in the beatmapset.
 
-Under "Tags" is where you can put the album title if you choose, along with specific subgenre information.
+#### Artist
 
-You should enter this information before you create a second difficulty, to ensure that the text will be consistent across all of the maps in the set.
+**The band, singer, composer, or group that performed or created the song.** Normally, this is automatically taken from the MP3 file's metadata, but double-check for proper spelling and punctuation.
 
--   **Artist:** **The band, singer, or group that performed or created the song.** osu! will pull the artist from the mp3 file, but please double check it to make sure it is spelled and punctuated correctly. **Anime/Game titles or companies do not belong here (they belong to Tags).** You can usually find the composer for anime and game songs on Wikipedia or other reliable sources such as [AnimeNewsNetwork](https://www.animenewsnetwork.com/) or [MyAnimeList](https://myanimelist.net/). Foreign names should be presented in the same name order as used on Wikipedia (generally, Given Name followed by Family Name, the western order, for consistency).
+*Notice: Anime or game titles belong in the [source](#source) field. Vocaloid names or company names belong in the [tags](#tags) field.*
```

Comment:
`source` -> `Source`
First occurrence, but should referencing fields be capitalized?

`Vocaloid names or company names` -> `Vocaloid or company names`?